v22.2/striim.md

```diff
@@ -0,0 +1,116 @@
+---
```

Comment:
Is this a new page? Looks like it, I think you need to add to the sidebar?

Reply:
🤦‍♂️ Added, thanks for the catch.